packages/peregrine/lib/Toasts/useToastContext.js

```diff
@@ -60,7 +60,8 @@ const reducer = (prevState = initialState, action = {}) => {
     }
 };
 
-const ToastContext = createContext();
+const ToastContext =
+    window.__ToastContext || (window.__ToastContext = createContext());
```

Comment:
Yeah, this line confuses me, especially since `window.__ToastContext` is not referenced anywhere else. If there's a bug with this particular context object, we should diagnose it and fix it.

Reply:
I don't know how this change made it in here: must have been a bad old rebase. Removed.